src/devices/bus/nes/multigame.cpp

```diff
@@ -3316,7 +3316,7 @@ void nes_bmc_ctc12in1_device::write_h(offs_t offset, u8 data)
 			break;
 	}
 
-	m_vram_protect == !BIT(m_reg[0], 7) || (offset & 0x6000) == 0x6000;
+	m_vram_protect = !BIT(m_reg[0], 7) || (offset & 0x6000) == 0x6000;
```

Comment:
My bad. Thanks! And cool, that seems to fix the dancing Pacman walls which I was confused about.

Reply:
Nice!  I only stumbled upon this because Clang whined.  I'm surprised that nobody else found this.

Reply:
Yeah, I'm surprised I didn't get a warning. Would you mind removing the TODO comment about 30 lines earlier? It seems it also fixes the problem with Legend of Kage's title screen. Thanks for finding it :)

Reply:
Done!

Reply:
Many thanks! Glad that turned out just to be my error and one that triggered a compiler complaint. I doubt anyone is going to investigate the handful of odd multicart TODOs otherwise :)

Reply:
Compilers are horribly inconsistent about warnings across targets/configurations.  One that hit me the other day was a number of GCC string format overflow warnings that only triggered on Linux and not MinGW.